METIS/headers/FITS_wcu_keywords.yaml

```diff
@@ -0,0 +1,23 @@
+- ext_type: PrimaryHDU
+  keywords:
+
+    HIERARCH:
+      ESO:
+        INS:
+          OPTI17:               # WCU periscope arm
+            ID: "WCU periscope"
+            NAME: "IN"
+          OPTI18:               # WCU PP1 masks
+            ID: "WCU PP1 masks"
+            NAME: "#pupil_masks.current_mask"
```

Comment:
Are you sure this is the proper use of `ID` and `NAME`?

From the filters I learned that `NAME` is a human-readable name for the filter, e.g. "SLOAN_r", which is similar to what you have, but that `ID` is a reference to a specific physical piece of hardware. E.g. a numerical value.

That is, say we have the same filter fabricated twice, and put one in the instrument and have the other as spare. If the one in the instrument will at some point be replaced by the spare, then the `NAME` stays the same, but the `ID` will change.

If that also applies to the other OPTI instruments, then I would expect something different in the `ID` column.

At least, that is my understanding, which could be wrong.

Reply:
Oh, I though `ID` was the identifier for the optical element (i.e. `OPTI18`), not for the mask itself. But what you say makes sense. Let me check some other instruments.

Reply:
`DESC` sounds good.